ui/jobs/components/mnk.js

```diff
@@ -48,34 +36,9 @@ export function setup(bars) {
         p.classList.remove('dim');
     }
 
-    // TODO: Remove bars.speedBuffs.lightningStacks,
-    // and change code to calculate speed by level in calcGCDFromStat function
-    // when cn/ko update 5.4
-    if (lightningTimer) {
-      bars.speedBuffs.lightningStacks = jobDetail.lightningStacks;
-      lightningTimer.fg = lightningFgColors[bars.speedBuffs.lightningStacks];
-      if (bars.speedBuffs.lightningStacks === 0) {
-        // Show sad red bar when you've lost all your pancakes.
-        lightningTimer.stylefill = 'fill';
-        lightningTimer.value = 0;
-        lightningTimer.duration = 0;
-      } else {
-        lightningTimer.stylefill = 'empty';
-
-        // Setting the duration resets the timer bar to 0, so set
-        // duration first before adjusting the value.
-        const old = parseFloat(lightningTimer.duration) - parseFloat(lightningTimer.elapsed);
-        const lightningSeconds = jobDetail.lightningMilliseconds / 1000.0;
-        if (lightningSeconds > old) {
-          lightningTimer.duration = 16;
-          lightningTimer.value = lightningSeconds;
-        }
-      }
-    } else {
-      // For now, we just assign bars.speedBuffs.lightningStacks
-      // as corresponding stacks via current level
-      bars.speedBuffs.lightningStacks = getLightningStacksViaLevel(bars.level);
-    }
+    // For now, we just assign bars.speedBuffs.lightningStacks
+    // as corresponding stacks via current level
```

Comment:
```suggestion
    // After the 5.4 changes, we just assign bars.speedBuffs.lightningStacks
    // as corresponding stacks via current level
```
I know it's not your comment, but I think it might age a little better this way.